front/tests/009-stdcm-between-two-trains.spec.ts

```diff
@@ -0,0 +1,153 @@
+      playwrightHomePage.page
+    );
+    await playwrightRollingstockModalPage.openRollingstockModal();
+
+    const rollingstockCard = playwrightRollingstockModalPage.getRollingstockCardByTestID(
+      VARIABLES.rollingstockTestID
+    );
+    await rollingstockCard.click();
+    await rollingstockCard.locator('button').click();
+
+    // ***************** Select Origin/Destination *****************
+    await scenarioPage.openTabByText('Itinéraire');
+    await playwrightMap.page.waitForTimeout(2000);
+    const itinerary = scenarioPage.getItineraryModule;
+    await expect(itinerary).toBeVisible();
+    await expect(scenarioPage.getMapModule).toBeVisible();
+
+    // Search and select origin
+    await playwrightMap.selectOrigin(
+      PATH_VARIABLES.originSearchDijon || PATH_VARIABLES.originSearch
+    );
+
+    // Search and select destination
+    await playwrightMap.selectDestination(
+      PATH_VARIABLES.destinationSearchMacon || PATH_VARIABLES.destinationSearch
+    );
+
+    await scenarioPage.checkPathfindingDistance(pathfindingDistance);
+
+    // ***************** Create train and edit it *****************
+
+    await scenarioPage.addTrainSchedule();
+    await scenarioPage.page.waitForSelector('.dots-loader', { state: 'hidden' });
+    await scenarioPage.checkToastSNCFTitle('Train ajouté');
+    await scenarioPage.returnSimulationResult();
+  });
+
+  // ***************** Test STDCM *****************
+
+  test('Testing stdcm', async ({ page }) => {
+    playwrightHomePage = new PlaywrightHomePage(page);
+    scenarioPage = new PlaywrightScenarioPage(page);
+    projectPage = new ProjectPage(page);
+    studyPage = new StudyPage(page);
+    playwrightSTDCMPage = new PlaywrightSTDCMPage(page);
+
+    await playwrightHomePage.goToHomePage();
+
+    await playwrightHomePage.goToSTDCMPage();
+    await playwrightSTDCMPage.getScenarioExploratorModalClose();
+    await playwrightSTDCMPage.openScenarioExplorator();
+    await playwrightSTDCMPage.getScenarioExploratorModalOpen();
+    await playwrightSTDCMPage.clickItemScenarioExploratorByName('_@Test integration project');
+    await playwrightSTDCMPage.clickItemScenarioExploratorByName('_@Test integration study');
+    await playwrightSTDCMPage.clickItemScenarioExploratorByName('_@Test integration scenario');
+
+    // ***************** Select Rolling Stock *****************
+    const playwrightRollingstockModalPage = new PlaywrightRollingstockModalPage(
+      playwrightHomePage.page
+    );
+    await playwrightRollingstockModalPage.openRollingstockModal();
+    const rollingstockCard =
+      playwrightRollingstockModalPage.getRollingstockCardByTestID(rollingStockTestID);
+    await rollingstockCard.click();
+    await rollingstockCard.locator('button').click();
+
+    // ***************** Select Origin/Destination *****************
+    const playwrightMap = new PlaywrightMap(playwrightHomePage.page);
+
+    // Search and select origin
```

Comment:
I think these comments are not useful anymore :)
```suggestion
```

Reply:
you're right, done